src/main/java/roomescape/domain/member/api/ApiMemberController.java

```diff
@@ -0,0 +1,56 @@
+package roomescape.domain.member.api;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseCookie;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.member.domain.Member;
+import roomescape.domain.member.error.exception.MemberErrorCode;
+import roomescape.domain.member.error.exception.MemberException;
+import roomescape.domain.member.service.MemberService;
+import roomescape.domain.member.service.dto.MemberLoginRequest;
+import roomescape.domain.member.service.dto.MemberResponse;
+
+@RestController
+@RequestMapping("/login")
+public class ApiMemberController {
+
+    private static final String TOKEN = "token";
+
+    public ApiMemberController(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    private final MemberService memberService;
+
+
+    @PostMapping
+    public ResponseEntity<?> login(@RequestBody MemberLoginRequest memberLoginRequest) {
+        String token = memberService.login(memberLoginRequest);
+        ResponseCookie responseCookie = ResponseCookie.from("token", token)
```

Comment:
```suggestion
        ResponseCookie responseCookie = ResponseCookie.from(TOKEN, token)
```

코드 전반적으로 상수를 많이 사용하시네요!
민욱님이 생각하시는 상수를 사용했을 때의 장점과 어떤 값을 상수화 하는지 그 기준에 대해 궁금해요.

예를 들어, 다른 코드를 보면 아래 값들도 상수화를 할거라 예상했는데요. 
그렇지 않아서 혼란스러웠습니다. ㅎㅎ
https://github.com/next-step/spring-roomescape-auth/blob/5ab7c8ccd8cae3b1af3ffc71988772967677d18f/src/main/java/roomescape/domain/member/service/MemberService.java#L27-L32

어떤 기준을 갖고 상수화를 하시나요?!